src/libraries/System.Net.Mail/tests/Unit/SmtpConnectionTests/EhloParseExtensionsTest.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace System.Net.Mail.Tests
 {
+    [PlatformSpecific(~TestPlatforms.Browser)]  // SmtpClient is not supported on Browser
```

Comment:
Just clarifying, does SmtpClient come into play because of SmtpConnection, that there is only a SmtpConnection when there is a SmtpClient?

Reply:
Yes